app/graph/mutations/explainer_item_mutations.rb

```diff
@@ -7,5 +7,18 @@ class Create < Mutations::CreateMutation
     argument :project_media_id, GraphQL::Types::Int, required: true
   end
 
+  class SendExplainersToPreviousRequests < Mutations::BaseMutation
+    argument :id, GraphQL::Types::Int, required: true
+    argument :range, GraphQL::Types::Int, required: true
```

Comment:
Please add a description or comment to make clear what's expected here - is it how many days ago the request was sent?

Reply:
Done